superset-frontend/src/explore/store.js

```diff
@@ -69,13 +69,15 @@ export function applyDefaultFormData(inputFormData) {
   const controlFormData = getFormDataFromControls(controlsState);
 
   const formData = {};
-  Object.keys(controlsState).forEach(controlName => {
-    if (inputFormData[controlName] === undefined) {
-      formData[controlName] = controlFormData[controlName];
-    } else {
-      formData[controlName] = inputFormData[controlName];
-    }
-  });
+  Object.keys(controlsState)
+    .concat(Object.keys(inputFormData))
```

Comment:
Dynamically imported plugins don't have any controls state available until after the plugin has loaded. This change makes it so that we use the fields from the chart's form data instead of only using the controls.